spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml')
```

Comment:
jeah but if you have no file at all, this will fail badly without an understandable message. Just an `no implicit conversion of nil into String` thrown by `File.exist?` for example